peru/parser.py

```diff
@@ -10,7 +10,7 @@
 from .scope import Scope
 
 
-DEFAULT_PERU_FILE_NAME = 'peru.yaml'
+DEFAULT_PERU_FILENAMES = ['peru.yaml', 'peru.yml', '.peru.yml', '.peru']
```

Comment:
`.peru` is the [default name of the state directory](https://github.com/buildinspace/peru/blob/8e915a8069fcec2c105dcc845c4d12c1db7aa96a/peru/runtime.py#L84) (apologies for not making that a more explicit constant). If you run `peru sync`, you'll notice the `.peru` directory gets created.